### DISCUSSION TITLE
Add tycoon park from RCT Classic to the extras tab in the scenario window

### DISCUSSION THREAD

**Discussion by ZeeMaji** [OpenRCT2/OpenRCT2#18110] [Feature requests] [answered]
Currently every other extra scenario from the original games is in this tab, i think it would be nice to include this one as well as it is indeed an extra scenario that comes with RCT Classic that'd fit well in this tab. Currently it is just lumped in with all of the other sub-category scenarios in the other tab.

**Comment by 733737** [comment]
I want to know how other people have more DLC parks than I do, I don't have RCT Classic linked, but still I don't have "gameplay" or "PC gamer" or some other ones that are supposedly official

**Reply by ZeeMaji** [reply]
Those were as their name suggests, downloadable parks. They were available on rct1/2's official websites back in the day.
The only ones that were included in any re-releases of the games are fort anachronism (comes with RCT1 deluxe) and panda world (comes with RCT Classic)
"gameplay", PC Player, and PC Gaming World were for some old magazines to review the game or something like that back in the rct1 days, so it makes sense that they were not included in later re-releases of the game due to the branding.
You should be able to find them on some rct archives if you look around.

**Reply by 733737** [reply]
why doesn't openrct2 come with gameplay et al? Aren't they of historical importance and royalty free?

**Reply by ZeeMaji** [reply]
I'm not sure, i'd say due to potential copyright issues but then again we include the rct2 dlc panda & pink water objects so maybe it's not entirely out of the question? Although these do seem a bit more complicated due to them being involved with non-rct brands.

**Reply by 733737** [reply]
what brands exactly? And are brands actually copyrighted? Some are trademarked maybe

**Comment by ZeeMaji** [comment, marked as answer]
Has been implemented by #18214
